### PR TITLE
Easy add superblock

### LINKED ISSUE #1543: Fix committee size adjustment during initial synchronization

### LINKED ISSUE #1544: GetSuperblockMessage to be used when syncing
The idea is that with the last consolidated superblock hash, peers can ask for the superblock associated so as to retrieve the correct committee size

### PR DESCRIPTION
**Pull request by tmpolaczyk**
Close #1543
Close #1544

### CODE REVIEW

File: data_structures/src/chain.rs [hunk 1]
```diff
@@ -594,7 +594,8 @@ pub struct BlockMerkleRoots {
 /// as of the last block in that period.
 /// This is needed to ensure that the security and trustlessness properties of Witnet will
 /// be relayed to bridges with other block chains.
-#[derive(Debug, Eq, PartialEq, Clone, Serialize, Deserialize, Default)]
+#[derive(Debug, Eq, PartialEq, Clone, Serialize, Deserialize, Default, ProtobufConvert)]
```
Comment (girazoki) [inline data_structures/src/chain.rs:597]: Order

File: data_structures/src/types.rs [hunk 2]
```diff
@@ -1,5 +1,6 @@
 use std::fmt;
 
+use crate::chain::SuperBlock;
```
Comment (girazoki) [inline data_structures/src/types.rs:3]: fmt

File: node/src/actors/chain_manager/handlers.rs [hunk 3]
```diff
@@ -1,6 +1,8 @@
 use actix::{fut::WrapFuture, prelude::*};
 use futures::future::Future;
-use std::{collections::BTreeMap, collections::HashSet, convert::TryFrom, net::SocketAddr};
+use std::{
+    collections::BTreeMap, collections::HashSet, convert::TryFrom, net::SocketAddr, time::Duration,
```
Comment (girazoki) [inline node/src/actors/chain_manager/handlers.rs:4]: fmt

File: node/src/actors/chain_manager/mod.rs [hunk 4]
```diff
@@ -1283,14 +1287,18 @@ impl ChainManager {
                     let reputed_ars = ARSIdentities::new(reputed_ars_members);
 
                     // Committee size should decrease if sufficient epochs have elapsed since last confirmed superblock
```
Comment (girazoki) [inline node/src/actors/chain_manager/mod.rs:1289]: lets add a comment specifying when we need to call the following function

**Comment by tmpolaczyk** [comment]
Superseded by #1557

**Pull request closed without merging**